bionic-arm64/Dockerfile

```diff
@@ -62,5 +47,4 @@ ENV AS=/usr/bin/aarch64-linux-gnu-as \
   CPP=/usr/bin/aarch64-linux-gnu-cpp-8 \
   CXX=/usr/bin/aarch64-linux-gnu-g++-8 \
   LD=/usr/bin/aarch64-linux-gnu-ld \
-  FC=/usr/bin/aarch64-linux-gnu-gfortran-8 \
```

Comment:
fyi, the only reason this was included is to match all the other dockerfiles that have the fortran compiler specified